### DISCUSSION TITLE
why are emmet & html a different extension

### DISCUSSION THREAD

**Discussion by JustFrederik** [zed-industries/zed#10984] [Feature Requests]
why are emmet and html different extension & why can't emmet be found when searching for html. I mean they are different language servers, but they are meant to be used together.
Both extensions are from zed directly & both are for html/css

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
